lms/envs/devstack_docker.py

```diff
@@ -31,3 +31,29 @@
     'ENABLE_COURSE_DISCOVERY': False,
     'ENABLE_DASHBOARD_SEARCH': False,
 })
+
+ENABLE_MKTG_SITE = os.environ.get('ENABLE_MKTG_SITE', False)
+MKTING_SITE_ROOT = os.environ.get('MKTING_SITE_ROOT', 'http://localhost:8080')
```

Comment:
The setting is actually called `MKTING_SITE_ROOT`? Not `MARKETING_SITE_ROOT`?

![](http://i3.kym-cdn.com/entries/icons/facebook/000/000/063/Picture_2_c.jpg)